kafka-connect-events/src/test/java/io/tabular/iceberg/connect/events/EventSerializationTest.java

```diff
@@ -65,9 +65,9 @@ public void testCommitResponseSerialization() {
     assertThat(payload.commitId()).isEqualTo(commitId);
     assertThat(payload.tableName().toIdentifier()).isEqualTo(TableIdentifier.parse("db.tbl"));
     assertThat(payload.dataFiles()).hasSize(2);
-    assertThat(payload.dataFiles()).allMatch(f -> f.specId() == 1);
+    assertThat(payload.dataFiles()).allMatch(f -> f.specId() == 0);
```

Comment:
for posterity's sake, this change is because the `createDataFile` function now creates files with unpartitioned spec.